app/controllers/api/auth/series_controller.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+class Api::Auth::SeriesController < Api::SeriesController
+
+  include ApiAuthenticated
+
+  api_versions :v1
+
+  filter_resources_by :account_id
+
+  represent_with Api::Auth::SeriesRepresenter
+
+  def sorted(res)
+    res.order('updated_at desc')
```

Comment:
This was just a fast way to get the sorted-ness I wanted.  But I'm willing to debate how we handle sorting.  Is there any existing query param?